mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_workloads.go

```diff
@@ -0,0 +1,177 @@
+package apphub
```

Comment:
Can you please add the test for this data source in the same folder?

Reply:
Done.